package it.csi.sigit.sigitwebn.business.dao.sigitwebn.exceptions;

/**
 * Le eccezioni rilanciate dai metodi del DAO sigitTConsumo14_4 sono di questo tipo.
 * @generated
 */
public class SigitTConsumo14_4DaoException extends DaoException {
	/** 
	 * @param message
	 * @generated
	 */
	public SigitTConsumo14_4DaoException(String message) {
		super(message);
	}

	/**
	 * @param message
	 * @param cause
	 * @generated
	 */
	public SigitTConsumo14_4DaoException(String message, Throwable cause) {
		super(message, cause);
	}

}
